whois-api/src/main/java/net/ripe/db/whois/api/rest/RestServiceHelper.java

```diff
@@ -173,7 +177,16 @@ public static WebApplicationException createWebApplicationException(final Runtim
         return new WebApplicationException(responseBuilder.build());
     }
 
+    public static void checkHttp(final HttpServletRequest request, final UpdateContext updateContext) {
+        final String header = request.getHeader(HttpHeader.X_FORWARDED_PROTO.toString());
+        if (!org.apache.commons.lang3.StringUtils.isEmpty(header) && HttpScheme.HTTP.is(header)){
```

Comment:
Can you not add warning in RestHelper but in service class for synupdate? This method should give us whether it is http or https, and rest of logic to throw error or warning should reside in controller